yarn.lock

```diff
@@ -2988,6 +2988,42 @@
   dependencies:
     "@types/node" ">= 8"
 
+"@ovh-ux/component-rollup-config@^8.0.0 || ^9.0.0":
```

Comment:
Looks weird it is added to yarn.lock . I think it is taking an older version available on registry, latest on monorepo is `10.0.2`